importer/DkBygningDa.py

```diff
@@ -50,8 +60,24 @@ def set_inception(self):
                 self.add_statement(
                     "inception", {"time_value": {"year": inception}})
 
+    def set_monuments_all_id(self):
+        """
+        Map which column name in specific table
+        is used as ID in monuments_all.
+        """
+        self.monuments_all_id = str(self.kommunenr) + "-" + \
```

Comment:
This would probably be more legible as "{kommun}-{ejendom}-{bygning}".format(...)